config.json

```diff
@@ -69,6 +69,16 @@
       ]
     },
     {
+      "uuid": "7eba3eas-8395-4lu6-ca21-7651s11ab4b4",
+      "slug": "perfect-numbers",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "Maybe"
```

Comment:
Number theory, yes, no?

Reply:
Yhea, sorry. I think I had this done before that.